CONTRIBUTING.rst

```diff
@@ -0,0 +1,158 @@
+Development Setup
+=================
+
+Cloning the project
+-------------------
+
+::
+
+    git clone git@github.com:SFDO-Tooling/MetaCI
+    cd MetaCI
+
+Making a virtual env
+--------------------
+
+MetaCI development requires Python v3.7. If ``which python3.7`` returns a
+non-empty path, it's already installed and you can continue to the next step. If
+it returns nothing, then install Python v3.7 using ``brew install python``, or
+from `Python.org`_.
+
+.. _Python.org: https://www.python.org/downloads/
+
+There are a variety of tools that let you set up environment variables
+temporarily for a particular "environment" or directory. We use
+`virtualenvwrapper`_. Assuming you're in the repo root, do the following to
+create a virtualenv (once you have `virtualenvwrapper`_ installed locally)::
+
+    mkvirtualenv metaci --python=$(which python3.7)
+    setvirtualenvproject
```

Comment:
virtualenvwrapper is what OddBird uses but I don't know if any of us are using it at SFDO, so I'm open to recommending a different approach here.

Reply:
No strong preference.

Reply:
We made it clear there's more than one way to do it, so I think this is fine.